DetourCrowd/Source/DetourCrowd.cpp

```diff
@@ -386,7 +386,7 @@ bool dtCrowd::init(const int maxAgents, const float maxAgentRadius, dtNavMesh* n
 	m_maxAgents = maxAgents;
 	m_maxAgentRadius = maxAgentRadius;
 
-	dtVset(m_ext, m_maxAgentRadius*2.0f,m_maxAgentRadius*1.5f,m_maxAgentRadius*2.0f);
+	dtVset(m_halfExtents, m_maxAgentRadius*2.0f,m_maxAgentRadius*1.5f,m_maxAgentRadius*2.0f);
```

Comment:
Is this a (preexisting) bug? This ought to be only `m_maxAgentRadius` for X and Z, as far as I can see.
cc @recastnavigation/write